sdk/spring/spring-cloud-azure-integration-tests/src/test/java/com/azure/spring/cloud/integration/tests/eventhubs/binder/EventHubsBinderSyncModeIT.java

```diff
@@ -53,12 +54,18 @@ Supplier<Flux<Message<String>>> supply(Sinks.Many<Message<String>> many) {
         @Bean
         Consumer<Message<String>> consume() {
             return message -> {
-                LOGGER.info("EventHubBinderRecordModeIT: New message received: '{}'", message.getPayload());
+                LOGGER.info("EventHubBinderSyncModeIT: New message received: '{}'", message.getPayload());
                 if (message.getPayload().equals(EventHubsBinderSyncModeIT.MESSAGE) && message.getHeaders().containsKey("x-opt-enqueued-time")) {
                     LATCH.countDown();
                 }
             };
         }
+
+        @ServiceActivator(inputChannel = "errorChannel")
+        public void receiveProduceErrorMsg(Message sendFailedMsg) {
```

Comment:
IIRC, this `errorChannel` is the global Spring Integration error channel, so maybe we can use a more general method name here, like `onError` or `processError`